### PR TITLE
Create Premium badge HTML element

### PR DESCRIPTION
**Pull request by jwitz**
For features included in Premium Astro plans.

Screen Shot 2022-12-15 at 5 37 27 PM

This is a v1, v2 will allow writers to pass an argument to the element so that it can appear either inline with text or after a header on its own line

### CODE REVIEW

File: astro/airflow-api.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ sidebar_label: 'Airflow REST API'
 id: airflow-api
```
Comment (jwitz) [inline astro/airflow-api.md:4]: Test doc

**Review by katiebrady-astro — APPROVED** [review]
Looks good to me! 🙌 I just changed the link to `/get-started/` since (I believe) that pricing page will be sunset soon in favor of just sending everyone to `/get-started/`. I also added a `title` to the image with the alt text so that'll show up if someone hovers with their mouse.
image

**Review by kwcanuck — APPROVED** [review]
Cool! I'll need to learn how to implement it. Should be straightforward.  I would truncate the tooltip to _An Astro Premium feature_ or _This feature is available only with Astro Premium_.

File: src/components/PremiumBadge/index.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Link from '@docusaurus/Link';
+import styles from './styles.module.css';
+
+export default function LinkCard() {
+  return (
+    <Link className={styles.premiumBadge} id="premiumBadge">
+        <div className={styles.premiumBadge}>
+        <a target="_blank" href="https://www.astronomer.io/get-started/"> 
+            <img src="/img/PremiumLarge.png" alt="This feature is an Astro Premium-tier service." title="This feature is an Astro Premium-tier service." />
```
Comment (jwitz) [inline src/components/PremiumBadge/index.js:10]:
```suggestion
            <img src="/img/PremiumLarge.png" alt="This feature is available only with the Astro Premium plan." title="This feature is available only with the Astro Premium plan." /> 
```

File: src/components/PremiumBadge/index.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Link from '@docusaurus/Link';
+import styles from './styles.module.css';
+
+export default function LinkCard() {
+  return (
+    <Link className={styles.premiumBadge} id="premiumBadge">
+        <div className={styles.premiumBadge}>
+        <a target="_blank" href="https://www.astronomer.io/get-started/"> 
+            <img src="/img/PremiumLarge.png" alt="This feature is available only with the Astro Premium plan." title="This feature is available only with the Astro Premium plan." />
```
Comment (jwitz) [inline src/components/PremiumBadge/index.js:10]:
```suggestion
            <img src="/img/PremiumLarge.png" alt="This feature is available only with an Astro Premium plan." title="This feature is available only with an Astro Premium plan." /> 
```

File: astro/airflow-api.md [hunk 4]
```diff
@@ -4,6 +4,8 @@ sidebar_label: 'Airflow REST API'
...
 ---
 
+import PremiumBadge from '@site/src/components/PremiumBadge';
+
```
Comment (jwitz) [inline astro/airflow-api.md:8]:
```suggestion
```

File: astro/airflow-api.md [hunk 5]
```diff
@@ -56,6 +56,8 @@ If you've configured a [CI/CD process](ci-cd.md) and you want to avoid generatin
 
 ### Retrieve the Deployment URL
 
+<PremiumBadge />
+
```
Comment (jwitz) [inline astro/airflow-api.md:60]:
```suggestion
```

**Comment by jwitz** [comment]
@kwcanuck To implement the badge, all you have to do is:

1. Add this line to the top of the doc:

`import PremiumBadge from '@site/src/components/PremiumBadge';`

2. After the topic header in a new line, add `<PremiumBadge />`

**Pull request merged**